docs/documentation/README.md

```diff
@@ -0,0 +1,4 @@
+# Documentation
+
+This directory contains only user specific documentation.
+Development documentation is located in the directory "contribute".
```

Comment:
Is it still fitting to use the term "directory" when talking about web-pages (even though they are represented through a directory structure underneath)?

Reply:
The idea was that the README.md files are files that describe the structure of the documentation and describe how the documentation can be build with jekyll. Therefore they don't start with [frontmatter](https://jekyllrb.com/docs/frontmatter/) and should not be rendered by jekyll.